src/leap/mx/mail_receiver.py

```diff
@@ -75,6 +76,11 @@ class MailReceiver(Service):
     """
     RETRY_DIR_WATCH_DELAY = 60 * 5  # 5 minutes
 
+    """
+    Number of days to keep stalled emails
+    """
+    BOUNCE_TIMEOUT_DAYS = 5
```

Comment:
matter of taste, but I was wondering if this would be more visible as a module-level constant.
thinkin on the possibilty of reading the value from a config file at some point...